universal-login-wallet/test/integration/pages/DashboardPage.ts

```diff
@@ -15,7 +15,7 @@ export default class DashboardPage {
   }
 
   async waitForHideModal() {
-    await waitForUI(this.wrapper, () => !this.wrapper.exists('.modal-wrapper'));
```

Comment:
why 4000?

Reply:
I had timeout error on my laptop (default is 3000 and test had time ~3300)